### PR TITLE
[onert] Add some ir::Graph tests

### PR DESCRIPTION
**Pull request by wateret**
- Add Input Output checker for Graph
- Add 1 pos and 3 neg tests

ONE-DCO-1.0-Signed-off-by: Hanjoung Lee <hanjoung.lee@samsung.com>

### CODE REVIEW

**Review by hseok-oh — APPROVED** [review]
LGTM

**Comment by wateret** [comment]
@nnfw-bot test cross-runtime-test-debug

**Comment by wateret** [comment]
@nnfw-bot test cross-runtime-test-debug

**Comment by hseok-oh** [comment]
```
15:28:31  [ RUN      ] Graph.neg_InvalidGraphFinish_BadInput
15:28:31  test_onert: /opt/jenkins_slave/workspace/nnfw/master/pr-cross-runtime-test-pipeline/runtime/onert/core/src/ir/Graph.cc:97: void onert::ir::Graph::finishBuilding(): Assertion `verifier::InputOutputChecker().verify(*this)' failed.
15:28:31  /jenkins/slave/workspace/nnfw/master/pr-cross-runtime-test-pipeline/Product/out/test/command/unittest: line 80: 11062 Aborted                 $TEST_BIN $(get_gtest_option)
15:28:31  Product/out/unittest_standalone/test_onert failed... return code: 134
```

File: tests/nnfw_api/src/one_op_tests/Add.cc [hunk 1]
```diff
@@ -122,10 +122,10 @@ TEST_F(GenModelTest, neg_OneOp_Add_InvalidShapeConst)
   std::vector<float> rhs_data{5, 4, 0, 7, 4, 0};
   uint32_t rhs_buf = cgen.addBuffer(rhs_data);
   int lhs = cgen.addTensor({{1, 2, 2, 1}, circle::TensorType::TensorType_FLOAT32});
-  int rhs = cgen.addTensor({{1, 2, 2, 1}, circle::TensorType::TensorType_FLOAT32, rhs_buf});
+  int rhs = cgen.addTensor({{1, 2, 3, 1}, circle::TensorType::TensorType_FLOAT32, rhs_buf});
   int out = cgen.addTensor({{1, 2, 3, 1}, circle::TensorType::TensorType_FLOAT32});
   cgen.addOperatorAdd({{lhs, rhs}, {out}}, circle::ActivationFunctionType_NONE);
-  cgen.setInputsAndOutputs({lhs, rhs}, {out});
+  cgen.setInputsAndOutputs({lhs}, {out});
```
Comment (wateret) [inline tests/nnfw_api/src/one_op_tests/Add.cc:128]: Not directly related to this commit, but introducing `InputOutputChecker` found a bug in the test case.

- `rhs` is a const so it shouldn't be an input
- If rhs and lhs shapes match, the current shape inference do not throw (it does not try to match output shape, just calculates new output shape)

**Review by hseok-oh — APPROVED** [review]
LGTM

**Pull request merged**